install/kubernetes/helm/istio/charts/pilot/templates/gateway.yaml

```diff
@@ -1,3 +1,58 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-autogenerated-k8s-ingress
+  namespace: istio-system
+spec:
+  selector:
+    istio: {{ .Values.global.ingressSelector }}
+  servers:
+  - port:
+      number: 80
+      protocol: HTTP
```

Comment:
http2

Reply:
Do we support http2 over plain text on gateway ?

Reply:
yep. Envoy can do both http2/http1. h2c I mean. and it auto detects. If you set this as Http, then some clusters end up not doing h2 upgrade and bad stuff happens